Python/bltinmodule.c

```diff
@@ -1591,8 +1591,12 @@ min_max(PyObject *args, PyObject *kwds, int op)
 
     if (positional)
```

Comment:
Now that we are modifying this code, add braces to the `if (positional)` per PEP7